.github/workflows/quickstart_ci.yml

```diff
@@ -93,11 +117,21 @@ jobs:
           cd ${{ inputs.QUICKSTART_PATH }}
           mvn -U -B -fae clean package -Popenshift
         shell: bash
+      - name: Run after script
+        env:
+          FILE: "./quickstarts/.github/workflows/quickstart_${{ inputs.QUICKSTART_PATH }}_ci_after.sh"
+        run: |
+          if test -f $FILE;
+            then
+              chmod +x $FILE 
+              bash $FILE
+          fi
+        shell: bash
       - uses: actions/upload-artifact@v3
         if: failure()
         with:
           name: surefire-reports-JDK${{ matrix.jdk }}-${{ matrix.os }}
-          path: 'quickstarts/$QUICKSTART_PATH/**/surefire-reports/*.txt'
+          path: 'quickstarts/${{ env.QUICKSTART_PATH }}/**/surefire-reports/*.txt'
```

Comment:
I **_think_** this should be `${{ inputs.QUICKSTART_PATH }}` but am not 100% sure!

Reply:
+1